datafusion/expr/src/tree_node/mod.rs

```diff
@@ -1,21 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-//
-//   http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-//! Tree node implementation for logical expr and logical plan
-
-pub mod expr;
```

Comment:
It makes little sense to arrange the LogicalPlan tree node in `datafuison/expr/src` rather than `datafusion/expr/src/logical_plan`

Reply:
This is also consistent with how physical plan and physical expr works

https://github.com/search?q=repo%3Aapache%2Farrow-datafusion%20tree_node.rs&type=code